### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-tf-module.git`](https://github.com/cisagov/skeleton-tf-module.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/cool-sharedservices-cdm.git cool-sharedservices-cdm
    cd cool-sharedservices-cdm
    git remote add skeleton https://github.com/cisagov/skeleton-tf-module.git
    git remote set-url --push skeleton no_push
    git switch develop
    git checkout -b lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add .github/CODEOWNERS README.md examples/basic_usage/README.md examples/basic_usage/main.tf examples/basic_usage/outputs.tf examples/basic_usage/variables.tf main.tf outputs.tf variables.tf versions.tf 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may _append_ to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Check the "Everything is cool" checkbox below:

    - [x] ✌️ The conflicts in this pull request have been resolved.

1. Mark this draft pull request "Ready for review".

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by mcdonnnj — APPROVED** [review]
LGTM ✅

**Review by dav3r — APPROVED** [review]
🧛

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍

File: versions.tf [hunk 1]
```diff
@@ -1,15 +1,19 @@
...
   # major version currently being used.  This practice will help us
   # avoid unwelcome surprises.
   required_providers {
-    # 3.43.0 is the first version of the Terraform AWS provider that
-    # correctly allows the local_ipv4_network_cidr and
-    # remote_ipv4_network_cidr keys of the aws_vpn_connection resource
-    # to be non-/32 CIDRs.
+    # Version 3.38.0 of the Terraform AWS provider is the first
+    # version to support default tags.
+    # https://www.hashicorp.com/blog/default-tags-in-the-terraform-aws-provider
```
Comment (mcdonnnj) [inline versions.tf]: Since we're no longer pinning on this version does this comment still make sense on its own?

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔ One question about a comment but it does not stop my approval.

File: versions.tf (hunk 1 above)
Comment (jsf9k) [reply versions.tf]: It's not on its own.  The comment about 3.43.0 follows.  I suspect I'm not understanding your point here.

Comment (dav3r) [reply versions.tf]: I think @mcdonnnj is saying that we no longer need the comment about `3.38.0`, since we are requiring `3.43.0` now.

Comment (mcdonnnj) [reply versions.tf]: Yes, I meant is the comment redundant since we are pinning a later version in this project.

Comment (jsf9k) [reply versions.tf]: See commit 5ef1301.

**Pull request merged**